stack.yaml

```diff
@@ -10,7 +10,7 @@ packages:
 system-ghc: true
 
 nix:
-  enable: true
+  enable: false
```

Comment:
This gives me compile errors when enabled. Not sure why.

Reply:
Not sure, both actually did work on my system though. What's the error?